administration/scheduling-and-retries.md

```diff
@@ -2,30 +2,39 @@
+[Fluent Bit](https://fluentbit.io) has an engine that helps to coordinate the data
+ingestion from input plugins. The engine calls the _scheduler_ to decide when it's time to
+flush the data through one or multiple output plugins. The scheduler flushes new data
+at a fixed time of seconds and retries when asked.
 
-Once an output plugin gets called to flush some data, after processing that data it can notify the Engine three possible return statuses:
+When an output plugin gets called to flush some data, after processing that data it
+can notify the engine using these possible return statuses:
 
-* OK
-* Retry
-* Error
+- `OK`: Data successfully processed and flushed.
+- `Retry`: If a retry is requested, the engine asks the scheduler to retry flushing
+  that data. The scheduler decides how many seconds to wait before retry.
+- `Error`: An unrecoverable error occurred and the engine shouldn't try to flush that data again.
 
-If the return status was **OK**, it means it was successfully able to process and flush the data. If it returned an **Error** status, it means that an unrecoverable error happened and the engine should not try to flush that data again. If a **Retry** was requested, the _Engine_ will ask the _Scheduler_ to retry to flush that data, the Scheduler will decide how many seconds to wait before that happens.
+## Configure wait time for retry
 
-## Configuring Wait Time for Retry
+The scheduler provides two configuration options called `scheduler.cap` and
+`scheduler.base` which can be set in the Service section. These determine the waiting
+time before a retry happens.
 
-The Scheduler provides two configuration options called **scheduler.cap** and **scheduler.base** which can be set in the Service section.
+| Key | Description | Default |
+| --- | ------------| --------------|
+| `scheduler.cap` | Set a maximum retry time in seconds. Supported in v1.8.7 or greater. | `2000` |
+| `scheduler.base` | Set a base of exponential backoff. Supported in v1.8.7 or greater. | `5` |
 
-| Key | Description | Default Value | 
-| -- | ------------| --------------| 
-| scheduler.cap | Set a maximum retry time in seconds. The property is supported from v1.8.7. | 2000 | 
-| scheduler.base | Set a base of exponential backoff. The property is supported from v1.8.7. | 5 |
+The `scheduler.base` determines the lower bound of time and the `scheduler.cap`
+determines the upper bound for each retry.
 
-These two configuration options determine the waiting time before a retry will happen. 
+Fluent Bit uses an exponential backoff and jitter algorithm to determine the waiting
+time before a retry. The waiting time is a random number between a configurable upper
+and lower bound. For a detailed explanation of the exponential backoff and jitter algorithm, see
```

Comment:
```suggestion
and lower bound. For a detailed explanation about the exponential backoff and jitter algorithm, see
```

Reply:
that one feels weird.
either "Detailed explanation of" 
or "more information about"